src/Makefile.am

```diff
@@ -841,8 +852,11 @@ nodist_libbitcoin_ipc_a_SOURCES = $(libbitcoin_ipc_mpgen_output)
 CLEANFILES += $(libbitcoin_ipc_mpgen_output)
 endif
 
+if BUILD_LEVELDB
 include Makefile.crc32c.include
```

Comment:
I don't understand why an explicit conditional is needed here. Shouldn't `make` be able to figure out for itself whether these libraries need to be built?

Reply:
> Shouldn't `make` be able to figure out for itself whether these libraries need to be built?

Yes, it should.

> I don't understand why an explicit conditional is needed here.

It is not _needed_. It's an optimization -- not processing unused `Makefile.*.include` files.

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/23345#issuecomment-968125192).

Reply:
Thanks. Yes, I don't think this was worth special-case optimizing, parsing performance of `make` is hardly an issue on modern systems afaik.